Automattic/jetpack
# RNA: change footer Jetpack symbol from font to SVG

#### Changes proposed in this Pull Request:
As Jetpack Component would be shared between plugins, it's better to use an embedded resource rather than using the Jetpack font residing only in Jetpack the plugin. The PR proposed a replacement of the Jetpack symbol from font to a SVG image. 

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Go to a site with Jetpack Instant Search subscription (enter URL directly /wp-admin/admin.php?page=jetpack-search&a8ctest)
* Ensure the footer looks good with both PC and mobile resolutions

![image](https://user-images.githubusercontent.com/1425433/125273278-dda6e500-e360-11eb-821f-86fe41ff6e1f.png)


cc @jwebbdev
